### PR TITLE
Release

### PR DESCRIPTION
**Pull request by stephanlindauer**
it's been a while since there was a release. sorry! :/
let's get this out and then i'll migrate the backend to aws.

### CODE REVIEW

File: app/src/main/java/de/stephanlindauer/criticalmaps/vo/Endpoints.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package de.stephanlindauer.criticalmaps.vo;
 
 public class Endpoints {
-    public static final String MAIN_POST = "http://api.criticalmaps.net/postv2";
-    public static final String IMAGE_POST = "http://api.criticalmaps.net/gallery/post.php";
-    public static final String GET_TWITTER = "http://api.criticalmaps.net/twitter/get.php";
+    public static final String MAIN_POST = "https://criticalmaps-api.stephanlindauer.de/postv2";
```
Comment (stephanlindauer) [inline app/src/main/java/de/stephanlindauer/criticalmaps/vo/Endpoints.java]: @headione can you also change the endpoints temporarily?

**Comment by cbalster** [comment]
Hey, I actually found the time to do something on the app today and basically have a PR ready - nothing special just some housekeeping/minor changes Can we maybe get that in before?

**Comment by stephanlindauer** [comment]
sure!

**Comment by cbalster** [comment]
This is ready to merge right?

**Comment by stephanlindauer** [comment]
currently testing it. but looks like it. :)

**Pull request merged**